justfile

```diff
@@ -267,6 +267,14 @@ compile-ios-ci target prod_flag="\"\"": _codegen-order-workaround
         {{just_executable()}} _compile-ios {{target}}
     fi
 
+flutter-run: dart-build
+    cd "$FLUTTER_WORKSPACE/example" && \
+        flutter run
+
+flutter-build target *args: dart-build
```

Comment:
For local development it would be really handy if we also automatically build the right rust libraries (depending on `target`). However, the recipe then no longer works on the CI :/ so we would also take that into account. 
same for flutter run
maybe we can create a ticket in jira to improve it later